plugins/module_utils/nutanix_api_client.py

```diff
@@ -172,3 +207,133 @@ def delete_image(image_uuid, client):
     response = client.request(
         api_endpoint="v3/images/{0}".format(image_uuid), method="DELETE", data=None)
     return response.json()["status"]["execution_context"]["task_uuid"]
+
+
+def get_cluster_uuid(cluster_name, client):
+    length = 250
```

Comment:
Can we use global vars to set default value
e.g. length = DEFAULT_LENGTH

Define global vars in some other file and import it or in same file is also fine.